### PR TITLE
Use the `shortHash` of the key instead of `toString()`

### PR DESCRIPTION
**Pull request by AlexV525**
https://github.com/flutter/flutter/issues/148983

### CODE REVIEW

**Comment by AlexV525** [comment]
@ujas-m-simformsolutions @himanshuGandhiSimform

File: lib/src/controllers/player_controller.dart [hunk 1]
```diff
@@ -31,10 +31,9 @@ class PlayerController extends ChangeNotifier {
   /// Provides [max] duration of currently provided audio file.
   int get maxDuration => _maxDuration;
 
-  final UniqueKey _playerKey = UniqueKey();
-
   /// An unique key string associated with [this] player only
-  String get playerKey => _playerKey.toString();
+  late final playerKey = shortHash(_playerKey);
```
Comment (vatsaltanna-simformsolutions) [inline lib/src/controllers/player_controller.dart:35]:
```suggestion
  final String playerKey = shortHash(UniqueKey());
```

Comment (vatsaltanna-simformsolutions) [reply lib/src/controllers/player_controller.dart:35]: We can remove the getter from here and directly use the variable.

Comment (AlexV525) [reply lib/src/controllers/player_controller.dart:35]: Yeah sure.

**Review by ujas-m-simformsolutions — APPROVED** [review]

**Comment by ujas-m-simformsolutions** [comment]
Thanks for the contribution @AlexV525 🤝

**Pull request merged**